apps/global/i18n/translations/pt.json

```diff
@@ -106,6 +106,11 @@
     "meetTheCollectiveTitle": "Conheça o Coletivo",
     "meetTheCollectiveDescription": "Dream On: Speaker Collective é uma equipe de autores, palestrantes e pregadores. Somos apaixonados por capacitar e ensinar os outros a perseguir seus sonhos e propósitos que Deus colocou na vida de cada um."
   },
+  "board": {
```

Comment:
You will need to update these translations to reflect the english changes here too